srcpkgs/newsflash/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'newsflash'
+pkgname=newsflash
+version=1.4.2
+revision=1
+wrksrc="news_flash_gtk-$version"
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo pkg-config glib-devel sqlite-devel gettext"
+makedepends="rust-std webkit2gtk-devel libhandy1-devel openssl-devel"
+short_desc="RSS feed reader that supports various web based services"
+maintainer="Michal Vasilek <michal@vasilek.cz>"
+license="GPL-3.0-only"
+homepage="https://gitlab.com/news-flash/news_flash_gtk"
+distfiles="https://gitlab.com/news-flash/news_flash_gtk/-/archive/$version/news_flash_gtk-$version.tar.gz"
+checksum=099318a6bc57e5e695352b2f3c7dd149c9b889f39cb9cbd58dc3f10f41d4948d
+
+pre_build() {
+	if [ -n "$CROSS_BUILD" ]; then
+		vsed -i build-aux/cargo.sh \
+			-e 's,"$CARGO_TARGET_DIR"/${TARGET}/news_flash_gtk,"$CARGO_TARGET_DIR"/'"${XBPS_CROSS_RUST_TARGET}/"'${TARGET}/news_flash_gtk,'
```

Comment:
Please use the same sed as https://github.com/void-linux/void-packages/pull/24218/files

Is the `CARGO_HOME` one not necessary?